core/src/test/java/org/jboss/jbossset/bugclerk/checks/PRAgainstProperBranch.java

```diff
@@ -72,6 +74,12 @@ private PullRequest mockPullRequest() {
         Codebase codebase = new Codebase("0.7");
         PullRequest mock = Mockito.mock(PullRequest.class);
         Mockito.when(mock.getCodebase()).thenReturn(codebase);
+        try {
+            Mockito.when(mock.getURL()).thenReturn(new URL("https://chingchong.com/test"));
+        } catch (MalformedURLException e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
```

Comment:
I would rather rethrow a "new IllegalArgumentException(e)"

Reply:
Yeah, but this wont fail, its a test with hardcoded URL which wont cause this :)